app/harvesters/loc_harvester.rb

```diff
@@ -0,0 +1,196 @@
+##
+# Harvester for Library of Congress
+# @see krikri::Harvester
+class LocHarvester < Krikri::Harvesters::ApiHarvester
+  include Krikri::Harvester
+
+  JSON_PARAM = '?fo=json'.freeze
+  
+  # @!attribute [r] opts
+  #   @return [Hash<Symbol, Object>] the options for the harvester
+  # @see .expected_opts
+  attr_reader :opts
+
+  ##
+  # @param opts [Hash] options for the harvester
+  #   The options for this harvester should be an array of URIs because LoC is
+  #   specifying sets of records to be harvested.
+  #
+  # Initial collections uris:
+  #   https://www.loc.gov/collections/american-revolutionary-war-maps/ (1,251 items)
+  #   https://www.loc.gov/collections/civil-war-maps/ (1,707 items)
+  #   https://www.loc.gov/collections/panoramic-maps/ (1,433 items)
+  #
+  # @see .expected_opts
+  def initialize(opts = {})
+    opts[:uri] = '' if opts[:api] # don't require a URI if we have api opts
+
+    opts[:name] ||= 'loc'.freeze
+    opts[:api]  ||= {}
+
+    opts[:api][:uris]          = Array.wrap(opts[:api][:uris])
+    opts[:api][:item_base_uri] = 'https://www.loc.gov/item/'.freeze
+
+    @http = Krikri::AsyncUriGetter.new(opts: {follow_redirects:  true,
+                                              inline_exceptions: true})
+    super
+  end
+
+  ##
+  # @return [Hash] A hash documenting the allowable options to pass to
+  #   initializers.
+  #
+  # @see Krikri::Harvester::expected_opts
+  def self.expected_opts
+    { 
+      key: :api,
+      opts: {
+        uris:          { type: :string, required: true,  multiple_ok: true},
+        item_base_uri: { type: :string, required: false, multiple_ok: false}
+      }
+    }
+  end
+
+  ##
+  # @param identifier [#to_s] the identifier of the record to get
+  # @return [#to_s] the record
+  def get_record(identifier)
+    build_record(request(build_record_uri(identifier)))
+  end
+
+  ##
+  # @see Harvester#record_ids
+  def record_ids
```

Comment:
The harvester interface has, in the past, assumed that the identifiers returned by `#record_ids` will be: 

1. the id that can be used in `#get_record` to retrieve the item.
1. the id used by the minter.

In LoCs case, it seems we want to use two different numbers for these purposes. I chose to keep the first constraint, and ignore the second. This ensures that code like this still works:

```ruby
my_harvester.record_ids.select { |id| ... }.map { |id| my_harvester.get_record(id) }
```